RichEditorView/Assets/editor/rich_editor.js

```diff
@@ -372,3 +373,67 @@ RE.getSelectedHref = function() {
 
     return href ? href : null;
 };
+
+/* Make sure all text nodes are wrapped in divs! */
+
+RE.wrapTextNodes = function() {
+    var contents = RE.editor.childNodes;
+    for (var i = 0; i < contents.length; i++) {
+        if (contents[i].nodeType === 3) {
```

Comment:
It might be nice to have an object or enum or something so we don't have magic numbers flying around. Maybe `nodeType == 3` is obvious to JS devs, but i have no idea.

Maybe `Node.TEXT_NODE` instead? https://developer.mozilla.org/en/docs/Web/API/Node/nodeType